README.md

```diff
@@ -135,7 +135,41 @@ git clone https://github.com/NVIDIA/cccl.git
 nvcc -Icccl/thrust -Icccl/libcudacxx/include -Icccl/cub main.cu -o main
 ```
 > **Note**
-> Use `-I` and not `-isystem` to avoid collisions with the CCCL headers implicitly included by `nvcc` from the CUDA Toolkit. All CCCL headers use `#pragma system_header` to ensure warnings will still be silenced as if using `-isystem`, see https://github.com/NVIDIA/cccl/issues/527 for more information. 
+> Use `-I` and not `-isystem` to avoid collisions with the CCCL headers implicitly included by `nvcc` from the CUDA Toolkit. All CCCL headers use `#pragma system_header` to ensure warnings will still be silenced as if using `-isystem`, see https://github.com/NVIDIA/cccl/issues/527 for more information.
+
+#### Conda
+
+CCCL also provides conda packages of each release via the `conda-forge` channel:
+
+```bash
+conda config --add channels conda-forge
+conda install cccl
+```
+
+This will install the latest CCCL to the conda environment's `include/` and `lib/cmake/` directories.
```

Comment:
Should we mention `$CONDA_PREFIX` here for clarity?
```suggestion
This will install the latest CCCL to the conda environment's `$CONDA_PREFIX/include/` and `$CONDA_PREFIX/lib/cmake/` directories.
```

Reply:
I debated this as well. I applied your suggestion, thanks!